source/assets/public/index.css

```diff
@@ -132,44 +146,55 @@ body {
     display: none;
   }
 
+  .breakpoint-phone .nav__room {
+    padding: 0;
+  }
+
   .nav--mobile__show {
-    padding: 6px;
-    position: fixed;
```

Comment:
I didn't think the mobile nav needed being fixed to the top since the pages really aren't that long.